UI/DesignWnd.cpp

```diff
@@ -905,39 +905,41 @@ namespace {
     /** A class to allow the storage of the state of a GUI availabilty filter
         and the querying of that state WRT a ship design. */
     class AvailabilityManager {
-    private:
-        // A tuple of the toogle state of the 3-tuple of coupled
-        // availability filters in the GUI:
-        // Obsolete, Available and Unavailable
-        std::tuple<bool, bool, bool> m_availabilities;
-
     public:
+        // DisplayedAvailabiltyType is indexed by Availability::Enum
+        using DisplayedAvailabilty = std::tuple<bool, bool, bool>;
```

Comment:
should probably be `DisplayedAvailabilities` ?

at least ility instead of ilty